client/app/core/dialog-field-refresh.service.js

```diff
@@ -2,35 +2,15 @@
 /* eslint angular/angularelement: "off" */
 
 /** @ngInject */
-export function DialogFieldRefreshFactory(CollectionsApi, EventNotifications) {
+export function DialogFieldRefreshFactory(CollectionsApi, EventNotifications, AutoRefresh) {
```

Comment:
If ya prepend _ to unused params, travis will shutup about em (that is if its required)